### ISSUE TITLE
Visual Studio build ModuleGenerator warning 

### ISSUE THREAD

**Issue by Waleed-KH** [microsoft/node-api-dotnet#285]
When building a .NET Node Module project in Visual Studio, it fail to generate the marshalling code

```
CSC : warning CS8032: An instance of analyzer Microsoft.JavaScript.NodeApi.Generator.ModuleGenerator cannot be created from C:\Users\...\.nuget\packages\microsoft.javascript.nodeapi.generator\0.7.8\analyzers\dotnet\cs\Microsoft.JavaScript.NodeApi.Generator.dll : Could not load file or assembly 'System.Runtime, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.
```

Visual Studio Version 17.9.6

**Comment by jasongin** [comment]
Yes, currently the generator targets .NET 6.0 or later, so it doesn't work with how VS runs MSBuild and the C# compiler on .NET Framework 4.x.

I am working on switching the generator to target .NET Standard 2.0, as is recommended for source-generators, but there are some issues I need to work through.

**Comment by Waleed-KH** [comment]
fixed #290
